llpc/lower/llpcSpirvLowerResourceCollect.h

```diff
@@ -68,15 +68,16 @@ class SpirvLowerResourceCollect : public LegacySpirvLower, public llvm::InstVisi
   bool detailUsageValid() { return m_detailUsageValid; }
 
   virtual bool runOnModule(llvm::Module &module);
-  void visitCalls(llvm::Module &module);
-  llvm::Value *findCallAndGetIndexValue(llvm::Module &module, llvm::CallInst *const targetCall);
 
   static char ID; // ID of this pass
 
 private:
   SpirvLowerResourceCollect(const SpirvLowerResourceCollect &) = delete;
   SpirvLowerResourceCollect &operator=(const SpirvLowerResourceCollect &) = delete;
 
+  void visitCalls(llvm::Module &module);
+  llvm::Value *findCallAndGetIndexValue(llvm::Module &module, llvm::CallInst *const targetCall);
```

Comment:
nit: the `const` can be dropped here. It only applies to the pointer, not to the pointee. This won't even change the function signature.

Reply:
Good point. The function always returns nullptr anyway (and has been for ~1 year) so I suggest to remove it entirely - now or in a pr where the class will be removed.